checkstyle.xml

```diff
@@ -67,7 +67,10 @@
         <module name="LineLength">
             <property name="max" value="120"/>
         </module>
-        <module name="MethodLength"/>
+	<module name="MethodLength">
```

Comment:
I need to increase the value slightly due to one of the changes in a specific protocol.  I did not want to spend the time breaking the decode() method into smaller pieces.  This may no longer be necessary given your other comments as this will reduce the code.